.github/workflows/test.yaml

```diff
@@ -15,9 +15,9 @@ jobs:
     steps:
 
     - name: Set up Go 1.x
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v4
       with:
-        go-version: ^1.18
+        go-version-file: "go.mod"
```

Comment:
I prefer this config because we often tend to forget to update language version on CI.
Please let me know if you don't.